data/sql/updates/pending_db_world/rev_1621427242858236100.sql

```diff
@@ -0,0 +1,79 @@
+UPDATE `item_template_locale` SET `name` = 'Diseño: Flecha de hoja de hielo' WHERE (`ID` = 52023) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Técnica: Glifo de Agua eterna' WHERE (`ID` = 50166) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Técnica: Glifo de descomposición presurosa' WHERE (`ID` = 50168) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Técnica: Glifo de Rejuvenecimiento rápido' WHERE (`ID` = 50167) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ametrino de luz trémula' WHERE (`ID` = 47012) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: brillo del otoño rígido' WHERE (`ID` = 41580) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ojo de dragón brillante' WHERE (`ID` = 42299) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ojo de dragón fracturado' WHERE (`ID` = 42303) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ojo de dragón grueso' WHERE (`ID` = 42315) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ojo de dragón lustroso' WHERE (`ID` = 42304) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ojo de dragón rúnico' WHERE (`ID` = 42309) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ojo de Zul vívido' WHERE (`ID` = 46899) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: rubí cárdeno fracturado' WHERE (`ID` = 46921) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: rubí cárdeno rúnico' WHERE (`ID` = 46916) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: rubí cárdeno sutil' WHERE (`ID` = 46922) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ópalo crepuscular enjundioso' WHERE (`ID` = 41702) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ópalo crepuscular real' WHERE (`ID` = 41701) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ópalo crepuscular regio' WHERE (`ID` = 41703) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ojo de dragón rápido' WHERE (`ID` = 42307) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: lágrima de pesadilla' WHERE (`ID` = 49112) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `Description` = 'Te enseña a tallar un ametrino de precisión.' WHERE (`ID` = 47010) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `Description` = 'Te enseña a tallar un ametrino destellante.' WHERE (`ID` = 47010) AND `locale` IN ('esMX','esES');
```

Comment:
you are overriding the texts, you should choose one of the two

Reply:
moreover you could merge the updates description and name in one line if you want, to have less code (less code usually means less possible errors)

Reply:
Thank you so much for noticing, ill fix it right away!